doc/content/microreactors/hpmr_h2/hpmr_h2_results.md

```diff
@@ -0,0 +1,64 @@
+# HPMR-H_2 Reactor Results
+
+*Contact: Stefano Terlizzi, Stefano.Terlizzi\@inl.gov , Vincent Labour&#233;, vincent.laboure\@inl.gov*
+
+## Neutronics Verification
+
+
+To verify the quality of the neutronics model, the Griffin and Serpent models are compared with a fixed temperature profiles ($T_f = T_m = 1000$K, $T_r = 800$K, and $c_H=1.8$). With  1 polar and 3 azimuthal angles per octant (24 angles total), the Griffin eigenvalue error is 259 pcm and the relative error in power distributions for each assembly is given in [table-floating ] below.
+
+
+!table id=table-floating     caption=Error of power computed from Griffin vs. reference Serpent calculations for 20 axial layers and assemblies.
+|	Axial layer	|	Assembly 1 |	Assembly 2 |	Assembly 3 |	Total |
+| --------------| ------------- | ------------- | ------------- | --------- |
+| 1 | 0 | 0 | 0 | 0 |
+| 2	        | 0	        | 0	        | 0	        | 0	    |
+| 3	        | 2.87%	    | 3.33% 	    | 2.40% 	    | 2.89% |
+| 4	        | -2.63%	    | -2.96%	    | -3.80%	    | -3.10% |
+| 5	        | 0.58%	    | -0.45%	    | -1.34%	    | -0.35% |
+| 6	        | 0.61%	    | -0.55%	    | -1.38%	    | -0.39% |
+| 7	        | 0.93%	    | -0.50%	    | -1.38%	    | -0.26% |
+| 8	        | 1.21%	    | -0.37%	    | -1.38%	    | -0.11% |
+| 9	        | 1.36%	    | -0.24%	    | -1.37%	    | -0.01% |
+| 10	        | 1.47%	    | -0.22%	    | -1.18%	    | 0.09% |
+| 11	        | 1.56%	    | -0.23%	    | -1.18%	    | 0.12% |
+| 12	        | 1.43%	    | -0.11%	    | -1.15%	    | 0.12% |
+| 13	        | 1.56%	    | -0.11%	    | -1.18%	    | 0.16% |
+| 14	        | 1.38%	    | -0.19%	    | -1.26%	    | 0.04% |
+| 15	        | 1.23%	    | -0.12%	    | -1.19%	    | 0.04% |
+| 16	        | 1.22%	    | -0.01%	    | -1.11%	    | 0.11% |
+| 18	        | 3.10%	    | 2.79%	    | 1.55%	    | 2.54% |
+| 19	        | 0	        | 0	        | 0	        | 0	    |
+| 20	        | 0	        | 0	        | 0	        | 0	    |
+| RMSE	    | 1.66%	    | 1.41%	    | 1.75%	    | 1.31% |
+
+In that table, Assembly 1 is any of the center fuel assembly and Assemblies 2 and 3 correspond to any peripheral fuel assembly surrounded by 4 and 3 fuel assemblies, respectively. The relative errors in power are shown for each axial 10-cm thick axial layer of each assembly type and for all of them combined (Total), with axial layers 1 and 20 corresponding to the bottom and top of the core, respectively. The values are 0 for the top and bottom two layers because they do not contain any fissile material. The last row reports the Root Mean Square (RMSE) of the error in the corresponding assembly types.
```

Comment:
```suggestion
In that table, Assembly 1 is any of the center fuel assembly and Assemblies 2 and 3 correspond to any peripheral fuel assembly surrounded by 4 and 3 fuel assemblies, respectively. The relative errors in power are shown for each 10-cm thick axial layer of each assembly type and for all of them combined (Total), with axial layers 1 and 20 corresponding to the bottom and top of the core, respectively. The values are 0 for the top and bottom two layers because they do not contain any fissile material. The last row reports the Root Mean Square (RMSE) of the error in the corresponding assembly types.
```